main.cc

```diff
@@ -288,6 +288,21 @@ static void tcp_syncookies_sanity() {
     }
 }
 
+static void tcp_timestamps_sanity() {
+    try {
+        auto f = file_desc::open("/proc/sys/net/ipv4/tcp_timestamps", O_RDONLY | O_CLOEXEC);
+        char buf[128] = {};
+        f.read(buf, 128);
+        if (sstring(buf) == "0\n") {
+            startlog.error("sysctl entry net.ipv4.tcp_timestamps is set to 0.\n"
+                          "Scylla requires to set net.ipv4.tcp_timestamps=1");
```

Comment:
changed to `sstring(buf) != "1\n"`

Reply:
I don't think that's what  @nyh - I think the check was good, but the error message was wrong - that it should be set to 1 or 2, just not 0.

Reply:
Hmm, @mykaul is probably right - "2" is also a legitimate option.
However, since I still never got an answer how exactly these tcp_timestamps are related to packet losses and/or Scylla problems, I have no idea if "2" (which is documented as: "Like 1, but without random offsets.") is ok or not ok...

Reply:
Exactly

Reply:
Reverted the check to `sstring(buf) == "0\n"`